assets/js/lists/add-to-list-button.js

```diff
@@ -4,9 +4,36 @@
 		bindings: {
 			selectedVariation: '<'
 		},
-        controller: ['accountApi', 'dialogService', function (accountApi, dialogService) {
+        controller: ['accountApi', 'dialogService', 'listsApi', 'customerService', function (accountApi, dialogService, listsApi, customerService) {
 			var $ctrl = this;
 			$ctrl.$onInit = function () {
+				compareProductInLists();
+			}
+
+			function compareProductInLists() {
+				$ctrl.buttonInvalid = true;
+                listsApi.searchLists({
```

Comment:
This call is redundant, you do not need to pass the list names to the getListsWithProduct API call